### DISCUSSION TITLE
Price point poll

### DISCUSSION THREAD

**Discussion by frodeheg** [frodeheg/no.sparegris#35] [Polls]
There was a request for adding price points per mode.
My first thought is to keep the interface simple so it doesn't overwhelm the users.
So if this is going to be added I would probably consider having an enable option for it under advanced settings which defaults to false. Consider this when answering the poll below.

Any other thoughts? Discuss...

**Comment by frodeheg** [comment]
Thinking some more about it... If you have a "visitor mode", maybe you don't care about prices and just want it warm and cozy regardless, so some kind of override would be nice. Although there is a different kind of override requested as well for servicing AC, I don't think I should implement that as part of the same override. (more on that in issue #30 )

Would this be a good way to control it:
* In case Price control has been activated:
   * Show a choice button on every mode that let you disable price control for this mode (not visible when price control is disabled)

Or is this better
* Same as above, but also with the choice "this mode has its own price points" - which enables different mode tabs under price point setup.

Or is this better:
* In advanced settings have a choice that affects all modes at once: Price points per mode yes or no - defaults to no.

Other suggestions?

**Reply by dooniem** [reply]
Not sure I understand the last option, but the one in the middle seems to keep it tidy, but sort of messy if you want to adjust more. I vote for that

**Reply by ugumba** [reply]
Personally, I don't think I could use this (and stay sane).
I can't see a use for a mode without price control, nor a mode with its own price controls.
